data/json/vehicleparts/rams.json

```diff
@@ -5,11 +5,20 @@
     "categories": [ "warfare" ],
     "location": "structure",
     "damage_modifier": 150,
+    "//": "constructed from a 1 meter armor sheet in most cases, affixed along two edges = 200cm weld to install.",
     "description": "A reinforced ram.  Place at the edge of vehicle to reduce damage taken in collisions, and to increase damaged delivered in collisions, if the ram is the first vehicle part to collide.",
     "requirements": {
-      "install": { "skills": [ [ "mechanics", 2 ] ], "time": "60 m", "using": [ [ "welding_standard", 5 ] ] },
-      "removal": { "skills": [ [ "mechanics", 2 ] ], "time": "30 m", "using": [ [ "vehicle_weld_removal", 1 ] ] },
-      "repair": { "skills": [ [ "mechanics", 3 ] ], "time": "60 m", "using": [ [ "repair_welding_standard", 1 ] ] }
+      "install": {
+        "skills": [ [ "mechanics", 2 ] ],
+        "time": "40 m",
+        "using": [ [ "welding_standard", 200 ], [ "vehicle_bolt_install", 2 ] ]
+      },
+      "removal": {
+        "skills": [ [ "mechanics", 2 ] ],
+        "time": "15 m",
+        "using": [ [ "vehicle_weld_removal", 1 ], [ "vehicle_wrench", 1 ] ]
```

Comment:
The `vehicle_wrench` requirement doesn't exist. Not sure why built-in tests don't catch this, but it broke the Hitchhiker's Guide 🙃

Reply:
https://github.com/CleverRaven/Cataclysm-DDA/pull/66168

Reply:
oh whoops lmao. thanks for fixing that